beacon-chain/core/blocks/block_operations_test.go

```diff
@@ -1287,6 +1287,105 @@ func TestConvertToIndexed_OK(t *testing.T) {
 	}
 }
 
+func TestVerifyIndexedAttestation_OK(t *testing.T) {
```

Comment:
Nice test!

Why not just do `helpers.ClearAllCaches()`? unless there's some caches you wanna preserve in between tests

Reply:
ah good catch, I copied the setup from another test, I will change that too